### DISCUSSION TITLE
what do you mean no downgrading 😭

### DISCUSSION THREAD

**Discussion by skoopy-star** [3kh0/ext-remover#179] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
why 😭

**Comment by ghost** [comment, marked as answer]
because half of you are immature and have no idea how any of this works/ what you're doing. PLUS, you will mess your record up. I'll delete and lock these posts until most of you can tell me exactly 1) what you're doing 2) why and 3) how these exploits work. It's concerning that you are willing to go this far just to bypass an extension
